README.md

```diff
@@ -1,56 +1,79 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# CHILLMOVIES
 
+> A movie website that displays movie from an Api and lets a user add reviews, likes and comment with a click.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS, JS
+- API
+- Kaban boards, Webpacks and Jests
+
+## Live Demo
 
-## Live Demo (if available)
+[Live Demo Link](https://maxwell011.github.io/Chill-Movies/)
 
-[Live Demo Link](https://livedemo.com)
+## Video Presentation
 
+[Video link](https://drive.google.com/file/d/1rxG4KAP-8vSpmqXKvwIuSPHUzvQWvE4m/view?usp=sharing)
 
```

Comment:
- [ ] I was expecting to see a piece of code (not just words) you are proud of according to this project's instructions
`The video contains a highlight about a piece of the code the student finds interesting or is proud of making.`. Kindly record another video, this time remember to highlight a piece of code you are proud of.